triton-vm/src/op_stack.rs

```diff
@@ -336,6 +338,24 @@ impl Display for OpStackElement {
     }
 }
 
+impl Add for OpStackElement {
+    type Output = Self;
+
+    fn add(self, rhs: Self) -> Self::Output {
+        let Ok(sum) = Self::try_from(self.index() + rhs.index()) else {
+            panic!("`OpStackElement` addition overflowed");
+        };
+        sum
+    }
+}
+
+impl WrappingAdd for OpStackElement {
+    fn wrapping_add(&self, &rhs: &Self) -> Self::Output {
+        let wrapped_sum = (self.index() + rhs.index()) % (OpStackElement::COUNT as u32);
+        wrapped_sum.try_into().unwrap()
+    }
+}
+
```

Comment:
This behavior used to be helpful for the instruction's previous iteration. I have left the functionality in but the only argument in favor of this code's existence is “it doesn't hurt to have it.” Let me know if you think it should be removed.